core/api/transport/session.hpp

```diff
@@ -68,8 +85,30 @@ namespace kagome::api {
      */
     virtual void respond(std::string_view message) = 0;
 
+    /**
+     * @brief makes `on close` notification to listener
+     * @param id session id
+     * @param type type of the closed session
+     */
+    void notify_on_close(SessionId id, SessionType type) {
```

Comment:
Naming, please.

Reply:
```notify_on_close``` seems normal.

Reply:
I mean, methods should be in camel case.